src/helpers/utility.js

```diff
@@ -101,24 +97,6 @@ export function gasToQtum(gas) {
   return new BigNumber(gas).multipliedBy(gasCostBN).toNumber();
 }
 
-/*
-* Returns the string formatted date and time based on a unix timestamp.
-* @param dateTime {Moment} A moment instance of the date and time to convert.
-* @return {String} Returns a formatted string.
-*/
-export function getLocalDateTimeString(unixSeconds) {
-  return moment.unix(unixSeconds).format(FORMAT_DATE_TIME);
-}
-
-export function getShortLocalDateTimeString(unixSeconds) {
-  const dateTime = moment.unix(unixSeconds);
-  if (moment.locale() === Zhlang.momentlocale) {
-    return dateTime.format(FORMAT_SHORT_DATE_TIME_CN);
```

Comment:
why not we change to 'LLL' here? so don't need to write moment.unix.  everywhere.